src/UI/TextField.elm

```diff
@@ -793,15 +795,26 @@ textFieldError : RenderConfig -> Maybe String -> Element msg -> Element msg
 textFieldError cfg errorCaption inputElement =
     case errorCaption of
         Just caption ->
-            Element.column
```

Comment:
What if, instead of adding the case here, in the `withError` you make it back to `Nothing` if the string is empty...